### DISCUSSION TITLE
Connecting Pybricks with OpenCV

### DISCUSSION THREAD

**Discussion by SaharshSS** [pybricks/support#1302] [Q&A]
I have both Pybricks and OpenCV running perfectly well in my VS Code. Is there any way to get them to run in one file and with a hub, that is, to create a realtime computer vision interaction?

They are both running through the same .venv terminal in a virtual environment.

**Comment by dlech** [comment]
You can use pybricksdev to communicate with the robot and opencv to communicate with the camper in the same program running on the computer.

**Reply by SaharshSS** [reply]
Could you please show me an example of this? I am using the command line version of pybricksdev and so I am unable to run an opencv file as that only runs through python.

**Reply by dlech** [reply]
There is an example program showing how to integrate pybricksdev and pygame at https://github.com/orgs/pybricks/discussions/1182#discussioncomment-6544876

You could use this as a starting point and replace the pygame bits with opencv.
